.github/actions/upload-gatk-test-results/action.yml

```diff
@@ -1,31 +1,33 @@
-name: 'upload-results'
-description: 'uploads our codeCov and jococo test results'
+name: 'upload-gatk-test-results'
+description: 'Uploads the results for matrix runs of gatk-tests.yml and maybe comments on failing runs on the PR associated with the test'
 inputs:
   warnPR:
-    description: 'Update the failure logs on the PR for this job'
-    required: true
-    default: 'false'
-  repo-token:
-    description: 'The GITHUB_TOKEN secret'
-    required: false
-  test-type:
-    description: 'Type to lable this test as'
-    required: true
-  jdk-version:
-    description: 'jdk used for this test'
+    description: 'If true this will add a comment about the jococo tests to the PR with a link to the failed test'
```

Comment:
Is this skipped by only artifact?

Reply:
yes. So the results are still there but they require a download. I will make a PR to make the commenting/reporting more robust for forked PRs.